crowbar_framework/app/controllers/application_controller.rb

```diff
@@ -24,6 +24,7 @@ class ApplicationController < ActionController::Base
   rescue_from Crowbar::Error::ChefOffline, with: :chef_is_offline
 
   before_filter :enable_profiler, if: Proc.new { ENV["ENABLE_PROFILER"] == "true" }
+  before_filter :force_installer, if: Proc.new { ENV["CROWBAR_MODE"] == "installer" }
```

Comment:
Style/Proc: Use `proc` instead of `Proc.new`. (https://github.com/bbatsov/ruby-style-guide#proc)